analyses/tmb-vaf-longitudinal/README.md

```diff
@@ -28,9 +28,25 @@ Then, we will leverage this information to create one column for the constant va
 
 `01-preprocess-data.Rmd` is a script written to process and generate the data to create corplots. (1) Calculate VAFs per each Kids_First_Biospecimen_ID, (2) Select paired longitudinal samples for the autopsy samples, and (3) Add information of number of timepoints and number of biospecimen samples per each patient case. This script generates `maf_autopsy.tsv` and `tmb_genomic.tsv` files for the next steps and are placed in `../../scratch`.
 
+`03-explore-TMB-mut.Rmd` is a script written to plot stacked barplots for mutation count per patient case and per biospecimen sample and timepoint.
+
+`function-create-barplot.R` is a script containing the functions to create (1) stacked barplots for all samples, and (2) barplots for each `Kids_First_Biospecimen_ID` in `Kids_First_Participant_ID`
+
+`function-create-dumbbell-plot.R` is a script containing the functions to create dumbbell plots for TMB and total number of mutations across timepoints and cancer types per patient case.
+
+
 ## Results
 
-There are 29 autopsy samples out of the total 119 patient cases (with genomic assays) with maf information. There are 61 (out of the 67) patient samples with TMB information. There are 44 biospecimen samples missing from both TMB and VAF files. 
+There are 29 autopsy samples out of the total 119 patient cases (with genomic assays) with maf information. There are 117 (out of the 118) patient samples with TMB information. There are 44 biospecimen samples missing from both TMB and VAF files. 
+
+
+Moreover, we explored TMB and the number of mutations across longitudinal samples with genomic assays. 
+
+Generally, Hypermutant TMB defined as ≤10 Mb, and Ultrahypermutant TMB defined as >100 mutations/Mb (https://pubmed.ncbi.nlm.nih.gov/29056344/). There are hyper mutant samples in the PBTA cohort. We excluded samples with >= 10 from downstream analysis. Attention is needed in cases with high number of mutations in only one timepoint as this will lead to un-matched longitudinal samples. We also remove those so we always have matched longitudinal samples.
```

Comment:
```suggestion
Generally, Hypermutant TMB defined as >=10 Mb, and Ultrahypermutant TMB defined as >100 mutations/Mb (https://pubmed.ncbi.nlm.nih.gov/29056344/). There are hyper mutant samples in the PBTA cohort. We excluded samples with >= 10 from downstream analysis. Attention is needed in cases with high number of mutations in only one timepoint as this will lead to un-matched longitudinal samples. We also remove those so we always have matched longitudinal samples.
```